app/models/activity.rb

```diff
@@ -4,6 +4,7 @@ class Activity < ApplicationRecord
   extend Mobility
 
   has_many_attached :activity_documents
+  has_and_belongs_to_many :tags
```

Comment:
```suggestion
  has_and_belongs_to_many :tags, optional: true
```

Is my assumption correct, that it's optional?